docker/flv2mp4-docker-image/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM centos:7
+
+MAINTAINER Hemani Katyal <hkatyal@redhat.com>
```

Comment:
Really happy to see this line (MAINTAINER ones), you've been abducted by Arquillian 😆